src/main/kotlin/de/bund/digitalservice/useid/events/Event.kt

```diff
@@ -0,0 +1,23 @@
+package de.bund.digitalservice.useid.events
+
+class Event(val consumerId: String, val encryptedRefreshAddress: String, val widgetSessionId: String) {
```

Comment:
I thought that `consumerId` and `widgetSessionId` are the same. Maybe we should talk about, who creates which id and when.

Reply:
@code28 I thought so too

Reply:
Good point. I will consolidate those two.